raiden/tests/scenarios/pfs7_multiple_payments.yaml

```diff
@@ -72,6 +76,10 @@ scenario:
             - assert: {from: 3, to: 2, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
             - assert: {from: 4, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
             - assert: {from: 3, to: 4, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "adding a wait before the asserts"
```

Comment:
You mean "after" the asserts, right? I really don't think there's any reason to have a wait here since no states are changed.